setup_rucio.py

```diff
@@ -113,6 +113,14 @@ def write_requirements():
         req_file.close()
 
 
+oracle_extras = ['cx_oracle>=5.1']
```

Comment:
is it necessary to give the versions here?

Reply:
necessary, I don't think. More recommended as sometimes the cx_oracle version are not strictly compatible. At some point, it has to be in sync with the versions used in CI/travis and in production

Reply:
The extras are only installed if you give this as a parameter on the pip install, correct?

Reply:
yes, it's one possibility:
    pip install rucio[oracle]

One additional is to have a meta package requiring this installation. like:
package rucio-oracle 
with:
install_requires = ["rucio[oracle]"]

specified in the setup.py

Reply:
for the versions, ok, as long as we don't force an old version, i think it's fine.

metapackages: would be good imho. i didn't know about this pip install module[extra] mechanism for example. it's not obvious i think.

Reply:
cf. http://peak.telecommunity.com/DevCenter/setuptools#declaring-extras-optional-features-with-their-own-dependencies